hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -282,7 +287,19 @@
         admin.mergeRegions(Bytes.toBytesBinary(left), Bytes.toBytesBinary(right), false);
       }
 %>    Merge request accepted. <%
-    } %>
+    } else if (action.equals("move")) {
+        Connection connect = master.getConnection();
+        Admin newAdmin = connect.getAdmin();
+        TableName tableName = TableName.valueOf(fqtn);
+        TableDescriptor td = TableDescriptorBuilder.newBuilder(newAdmin.getDescriptor(tableName))
+              .setRegionServerGroup(key).build();
+            newAdmin.modifyTable(td);
+        RSGroupAdminClient rsGroupAdminClient = new RSGroupAdminClient(connect);
```

Comment:
On master branch we do not need this any more.

Can do this on branch-2 backport?